tests/dvbuilder/dv_builder.go

```diff
@@ -0,0 +1,119 @@
+package dvbuilder
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+)
+
+const (
+	dvRandomNameLength       = 12
+	PVCSizeForRegistryImport = "512Mi"
```

Comment:
This is only valid for cirros image right? I think for fedora you would need a larger size

Reply:
Thanks for the review @alicefr. 

That was hard coded in the previous version. For now, I want only to preserve the same logic, but not adding new one.

Reply:
It should not be here in dvbuilder. This size should be passed from caller. For example: To be clear and easy to understand the `func NewBlockDataVolumeWithRegistryImport` should take imageUrl and imageSize as first two params. 

If you want to preserve the same logic, then make it a const in datavolume.go.  It can be named `SmallPVC` or `PVCSizeForRegistryImport`, and should be commented telling us why it is 512Mi.

Reply:
Now I am sure this size can go into containerdisk.go and be called `CirrosVolumeSize`